processing/src/main/java/io/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -180,7 +179,7 @@ public int getIndex(String value)
           // Return -2 for non-null values to match what the BitmapIndex implementation in BitmapIndexColumnPartSupplier
           // would return for getIndex() when there is only a single index, for the null value.
           // i.e., return an 'insertion point' of 1 for non-null values (see BitmapIndex interface)
-          return Strings.isNullOrEmpty(value) ? 0 : -2;
+          return value == null ? 0 : -2;
```

Comment:
looking at `BitmapIndex.java` i can see 
```java
public interface BitmapIndex
{
  public int getCardinality();

  public String getValue(int index);

  public boolean hasNulls();
```
can you please reconcile old and new APIs

Reply:
done.